### PR TITLE
Adding Changes for Tier One Requirements for JWST Community Software Products

### PR DESCRIPTION
**Pull request by mfixstsci**
NOT READY FOR REVIEW YET!

All requirements are here: https://github.com/spacetelescope/ins-jwst-community-software/blob/master/tier1_standards/README.md

Most of these requirements are met already just by hosting the code via github and having installation instructions in the `README`. @pontoppi do you know if the `master` branch is protected?

### CODE REVIEW

**Comment by mfixstsci** [comment]
Went ahead and uploaded the package to the test PyPI server:

https://test.pypi.org/project/jwst-backgrounds/1.1.1/

**Comment by mfixstsci** [comment]
@pontoppi we should almost be ready to go. Can you do me a favor and try the pip install command provided in the link above? If you don't have a problem installing the software I can push it up to the actual PyPI server.

**Comment by mfixstsci** [comment]
https://travis-ci.com/spacetelescope/jwst_backgrounds/builds/140305504

First `travis` build was a success!

**Comment by mfixstsci** [comment]
Does anyone want to take a look at this?! I will leave it up for review until Thursday but if I don't get any feedback, I will merge to meet the deadline.

**Comment by pontoppi** [comment]
Looks good to me. Thanks!

**Comment by bourque** [comment]
@mfixstsci I also made you an admin of the repository, so you should be able to implement branch rules now.

**Comment by mfixstsci** [comment]
Hey @bourque -- Ugh, okay maybe it's something with my conda environment. Let me start from scratch with your recipes and see what I can come up with.

File: setup.py [hunk 1]
```diff
@@ -11,7 +11,11 @@
       download_url = 'https://github.com/spacetelescope/jwst_backgrounds/',
       packages=['jwst_backgrounds'],
       package_data={'jwst_backgrounds': ['refdata/*.csv','refdata/*.txt','agreement/*.pdf']},
-      install_requires=['healpy>=1.10'],
+      install_requires=['healpy>=1.10',
+                        'healpy>=1.10',
```
[the remark this answers is not in the document]
Comment (mfixstsci) [reply setup.py]: @bourque got it, I went ahead and removed the duplicate from the list.

**Comment by mfixstsci** [comment]
@bourque did you try this by chance?

`from jwst_backgrounds import jbt`

The installs work fine. I am getting an import error from `healpy` when I try import this function as outlined in the documentation.

**Comment by bourque** [comment]
@mfixstsci Hmmm, no, I didn't actually try importing anything 🤔

**Comment by mfixstsci** [comment]
@bourque can you try it out real quick? It might be out of the scope for the software standards, but just thought I'd test this while I was at it.

**Comment by bourque** [comment]
@mfix I just tried:

```
conda create -n jwst-backgrounds-3.7 python=3.7
conda activate jwst-backgrounds-3.7
pip install jwst_backgrounds
python
from jwst_backgrounds import jbt
```

and everything worked fine.

However, I tried the same thing using python 3.6 and was not able to `pip install jwst_backgrounds`, getting the following error:

```
Traceback (most recent call last):
      File "/private/var/folders/6v/lt8sgkb54g976t19c4lgd6bw0004tc/T/pip-install-3xhdnszc/healpy/setup.py", line 242, in build_library
        build_args = self.pkgconfig(pkg_config_name)
      File "/private/var/folders/6v/lt8sgkb54g976t19c4lgd6bw0004tc/T/pip-install-3xhdnszc/healpy/setup.py", line 207, in pkgconfig
        token[index:].decode() for token in check_output(cmd, env=env).split()
      File "/Users/bourque/miniconda3/envs/jwst-backgrounds-3.6/lib/python3.6/subprocess.py", line 336, in check_output
        **kwargs).stdout
      File "/Users/bourque/miniconda3/envs/jwst-backgrounds-3.6/lib/python3.6/subprocess.py", line 418, in run
        output=stdout, stderr=stderr)
    subprocess.CalledProcessError: Command '('pkg-config', '--cflags-only-I', 'healpix_cxx >= 3.60.0')' returned non-zero exit status 1.

    During handling of the above exception, another exception occurred:

    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/6v/lt8sgkb54g976t19c4lgd6bw0004tc/T/pip-install-3xhdnszc/healpy/setup.py", line 540, in <module>
        scripts=['bin/healpy_get_wmap_maps.sh'],
      File "/Users/bourque/miniconda3/envs/jwst-backgrounds-3.6/lib/python3.6/site-packages/setuptools/__init__.py", line 145, in setup
        return distutils.core.setup(**attrs)
      File "/Users/bourque/miniconda3/envs/jwst-backgrounds-3.6/lib/python3.6/distutils/core.py", line 148, in setup
        dist.run_commands()
      File "/Users/bourque/miniconda3/envs/jwst-backgrounds-3.6/lib/python3.6/distutils/dist.py", line 955, in run_commands
        self.run_command(cmd)
      File "/Users/bourque/miniconda3/envs/jwst-backgrounds-3.6/lib/python3.6/distutils/dist.py", line 974, in run_command
        cmd_obj.run()
      File "/Users/bourque/miniconda3/envs/jwst-backgrounds-3.6/lib/python3.6/site-packages/setuptools/command/install.py", line 61, in run
        return orig.install.run(self)
      File "/Users/bourque/miniconda3/envs/jwst-backgrounds-3.6/lib/python3.6/distutils/command/install.py", line 545, in run
        self.run_command('build')
      File "/Users/bourque/miniconda3/envs/jwst-backgrounds-3.6/lib/python3.6/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/Users/bourque/miniconda3/envs/jwst-backgrounds-3.6/lib/python3.6/distutils/dist.py", line 974, in run_command
        cmd_obj.run()
      File "/Users/bourque/miniconda3/envs/jwst-backgrounds-3.6/lib/python3.6/distutils/command/build.py", line 135, in run
        self.run_command(cmd_name)
      File "/Users/bourque/miniconda3/envs/jwst-backgrounds-3.6/lib/python3.6/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/Users/bourque/miniconda3/envs/jwst-backgrounds-3.6/lib/python3.6/distutils/dist.py", line 974, in run_command
        cmd_obj.run()
      File "/Users/bourque/miniconda3/envs/jwst-backgrounds-3.6/lib/python3.6/distutils/command/build_clib.py", line 115, in run
        self.build_libraries(self.libraries)
      File "/private/var/folders/6v/lt8sgkb54g976t19c4lgd6bw0004tc/T/pip-install-3xhdnszc/healpy/setup.py", line 372, in build_libraries
        for key, value in self.build_library(lib_name, **build_info).items():
      File "/private/var/folders/6v/lt8sgkb54g976t19c4lgd6bw0004tc/T/pip-install-3xhdnszc/healpy/setup.py", line 303, in build_library
        check_call(cmd, cwd=build_temp, env=env)
      File "/Users/bourque/miniconda3/envs/jwst-backgrounds-3.6/lib/python3.6/subprocess.py", line 291, in check_call
        raise CalledProcessError(retcode, cmd)
    subprocess.CalledProcessError: Command '['make', 'install']' returned non-zero exit status 2.
```

**Comment by mfixstsci** [comment]
@bourque -- okay then there is definately something going on with my conda environment, thanks for checking. As for the python 3.6 issue, eeeeck. I will look into that.

**Comment by mfixstsci** [comment]
@bourque I just went ahead and did the following

```
conda create -n jwst-backgrounds-3.6 python=3.6

**installs**

conda activate jwst-backgrounds-3.6

pip install jwst_backgrounds
```

And everything worked fine... Am I missing something?!

**Comment by bourque** [comment]
@mfix Hmm, I just tried the same commands and got the same error as before 🤔 🤷‍♂

**Comment by mfixstsci** [comment]
Weird @bourque, should we just merge? I see filed issues with `healpy` etc and then work on these issues behind the scenes?

**Comment by bourque** [comment]
@mfixstsci Yeah I think we are fine to merge.  The problem seems to exist with `healpy` and is beyond the scope of these Tier 1 standards.  I would suggest to open a separate issue about this and try to get things working in the meantime.

**Comment by mfixstsci** [comment]
@bourque send it

**Comment by bourque** [comment]
@mfixstsci I think you just need to push the fix that removes the redundant `healpy` dependency

**Comment by mfixstsci** [comment]
@bourque cool, it said I did that yesterday

https://github.com/spacetelescope/jwst_backgrounds/pull/7/commits/51d8c89737deeb3fd7e2952d4aa7da999bc6750f

So if you are ready, I'd say go for it

**Pull request merged**